runtime/src/main/resources/com/sun/org/apache/xml/internal/serializer/output_xml.properties

```diff
@@ -0,0 +1,47 @@
+#
```

Comment:
This file is a fix for GraalVM not being able to properly discover properties files nested in other jars.